modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-forwarding.adoc
+
+[id="cluster-logging-log-forwarding-about_{context}"]
+= Understanding cluster log forwarding
+
+The {product-title} cluster log forwarding feature uses a combination of _outputs_ and _pipelines_ defined in the Log Forwarding Custom Resource to send logs to specific endpoints inside and outside of your {product-title} cluster. 
+
+An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+
+An output can be either:
+
+* `elasticsearch` to forward logs to an external Elasticsearch v5.x cluster, specified by server name or FQDN.
+* `forward` to forward logs to an external log aggregation solution. This option uses the Fluentd *forward* plug-in.
+
+A pipeline associates the type of data to an output. A type of data you can forward is one of the following:
+
+* `logs.app` - Container logs generated by user applications running in the cluster, except infrastructure container applications.
+* `logs.infra` - Logs generated by both infrastructure components running in the cluster and {product-title} nodes, such as journal logs. Infrastructure components are applications run on pods in the `openshift*`, `kube*`, or `default` projects.
+* `logs.audit` - Logs generated by the node Audit system (auditd), which are stored in the  */var/log/audit/audit.log* file, and the audit logs from the Kubernetes apiserver and the OpenShift apiserver. 
+
+Note the following:
+
+* Log forwarding does not provide secure storage for audit logs. You are responsible to ensure the endpoint is compliant with your organizational and governmental regulations and is properly secured. {product-title} cluster logging does not comply with those regulations.
+
+* An output supports TLS communication using a secret. Secrets must have keys of: *tls.crt*, *tls.key*, and *ca-bundler.crt* which point to the respective certificates for which they represent. Secrets shall have the key *shared_key* for use when using forward in a secure manner.
+
+* You are responsible to create and maintain any additional configurations that external destinations might require, such as keys and secrets, service accounts, port opening, or global proxy configuration.
+
+* Enabling the log forwarding feature disables the default log forwarding pipelines.
+
+The following example creates three outputs: 
+
+* the cluster logging managed Elasticsearch instance, 
```

Comment:
How do you configure the output of `cluster logging managed Elasticsearch instance` ?

Reply:
This output is configured for you if you do not configure LF.

Reply:
But in this example, it didn't clarify it. It says `The following example creates three outputs: `, one is `the cluster logging managed Elasticsearch instance, `, and there has 
```
Three pipelines send:
 the application logs to the cluter-logging-managed Elasticsearch and the *forward* plug-in, 
``` 
It may makes customer to think they need to configure pipelines and output for the clusterlogging managed elasticsearch.  @mburke5678 could you please make some changes per @jcantrill  ?

Reply:
@QiaolingTang @jcantrill There are two use cases here?

1. The user can send one of the outputs to the cluster-logging-managed Elasticsearch using `endpoint: elasticsearch.svc.messaging.cluster.local` and the other outputs to other endpoints? 
2. The user wants all outputs to go to the cluster-logging-managed Elasticsearch, the user does not configure any outputs and pipelines. 

If so, my example was intended to show case 1. 

I added a note at the top of the page to address case 2. 

See commit fee5f971a68c31560d634ad01176f58f3b70d89d

Reply:
@mburke5678, if user only want to send logs to the cluster-logging-managed elasticsearch, then no need to create the LF instance. 
If user create the LF instance, then the logs will be sent to the outputs defined in the LF instance. But I don't know if we officially support forwarding logs to cluster-logging-managed elasticsearch and user-managed-log-store at the same time. In my testing, it's OK to do that, but user may encounter some bugs, for example: https://url.corp.redhat.com/dc7d985
BTW, the endpoint of the cluster-logging-managed elasticsearch is `elasticsearch.openshift-logging.svc:9200`.

Reply:
> @mburke5678, if user only want to send logs to the cluster-logging-managed elasticsearch, then no need to create the LF instance.
> If user create the LF instance, then the logs will be sent to the outputs defined in the LF instance. But I don't know if we officially support forwarding logs to cluster-logging-managed elasticsearch and user-managed-log-store using LF at the same time. In my testing, it's OK to do that, but user may encounter some bugs, for example: https://url.corp.redhat.com/dc7d985. So I'm not sure if we should document about configuring output to the cluster-logging-managed elasticsearch in the LF instance.

We do and we should.  Ideally we would simply advise user's to create an output to the managed ES, but we are trying to support the upgrade scenario.  

> BTW, the endpoint of the cluster-logging-managed elasticsearch is `elasticsearch.openshift-logging.svc:9200`.

Reply:
@QiaolingTang I updated the cluster-logging ES endpoint. 
@jcantrill What is the concern with the upgrade scenario?

Reply:
There is no concern with the upgrade scenario.  I was trying to point out we need to support that scenario and thus the reason you opt into LF but if you already have CL deployed then it is supposed to just "work" and its using LF under the hood